env.go

```diff
@@ -83,7 +85,7 @@ func colorProfile(isatty bool, env environ) (p Profile) {
 		if p > Ascii {
 			p = Ascii
 		}
-		return
+		return p
```

Comment:
if we're doing this, we should also change it in line 73 to return `Profile` instead of `p Profile`